modules/sb-about-template-service-broker.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// * applications/service_brokers/installing-template-service-broker.adoc
+
+[id='sb-about-template-service-broker-{context}']
+= About the template service broker
+
+The _template service broker_ gives the service catalog visibility into the
+default Instant App and Quickstart templates that have shipped with
+{product-title} since its initial release. The template service broker can also
+make available as a service anything for which an {product-title} template has
+been written, whether provided by Red Hat, a cluster administrator or user, or a
+third party vendor.
+
+By default, the template service broker shows the objects that are globally
```

Comment:
@johnkim76 is this true? Can the operator tell the broker to watch a different namespace? 

@johnkim76 Can you look at the rest of this section and tell me what you think?